.travis.yml

```diff
@@ -15,13 +18,16 @@ notifications:
     on_failure: always  # options: [always|never|change] default: always
     on_start: never     # options: [always|never|change] default: always
 
+branches:
+  only:
+    - master
+
 before_script:
   - echo MAVEN_OPTS=-Xmx1536m > ~/.mavenrc
 
 # Enable integration tests
-# skip tests as they need an environment to run (SSO and Gateway urls)
 script:
-  - mvn verify -B -Pdependency-check -DskipTests
+  - mvn verify -B -DskipITs=false
```

Comment:
Acceptance tests should run only in Jenkins-x as it's building an environment to run the test against. Currently, no environment is available to run acceptance tests on Travis.

Reply:
this file was already there, I just removed the missing profile, so they might have been running anyway, shall we remove it and use just Jenkins? maybe it's good to review if we can just use a single CI

Reply:
The problem here is that you are enabling integration tests with `-DskipITs=false` (they were disabled before for Travis). However, I think we can completely remove Travis here and keep only Jenkins as you are suggesting. Travis is only checking compilations errors (what is already checked by Jenkins anyway).

Reply:
put -DskipITs back